cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,91 @@
+import { Calculator } from '/src/js/calculator';
+
+describe('계산기 테스트', () => {
+  before('기능', () => {
+    cy.visit('/');
+  })
+
+  it('더하기', () => {
+    clickDigit('2');
+    clickOperator('+');
+    clickDigit('5');
+    clickOperator('=');
+    cy.get('#total').invoke('text').should('eq', String('7'));
```

Comment:
```suggestion
    cy.get('#total').invoke('text').should('eq', '7');
```
어차피 문자열이기 때문에 이렇게만 표현해도 좋을 것 같아요!